cmd/thanos/query_frontend.go

```diff
@@ -193,7 +193,7 @@ func runQueryFrontend(
 	roundTripper = tripperWare(roundTripper)
 
 	// Create the query frontend transport.
-	handler := transport.NewHandler(*cfg.CortexHandlerConfig, roundTripper, logger)
+	handler := transport.NewHandler(*cfg.CortexHandlerConfig, roundTripper, logger, nil)
```

Comment:
No need to pass the Prometheus registerer because it's just used for query statistics, which is currently a Cortex specific thing.